src/unix/linux.c

```diff
@@ -1155,6 +1155,12 @@ static void uv__poll_io_uring(uv_loop_t* loop, struct uv__iou* iou) {
     uv__req_unregister(loop, req);
     iou->in_flight--;
 
+    /* if the op is not supported by the kernel retry using the threadpool */
+    if (e->res == -EOPNOTSUPP) {
```

Comment:
Just want to double check that this is the correct value for us to check. The specific error returned in https://github.com/nodejs/node/issues/50876 is `ENOTSUP`.  But that conflicts with the documentation (https://unixism.net/loti/ref-iouring/io_uring_register.html) where it says that it's supposed to return `EOPNOTSUPP`.

Though all this might not matter since on https://man7.org/linux/man-pages/man3/errno.3.html it says:

> `ENOTSUP` and `EOPNOTSUPP` have the same value on Linux, but according to POSIX.1 these error values should be distinct.

So just want to verify that checking `EOPNOTSUPP` will work even though the standard says the two error values are supposed to be different. Since in Linux they are the same anyways.

Reply:
I think the correct way is using `EOPNOTSUPP`. Nodejs is printing `ENOTSUP` because of the way `errno_string()` is [implemented](https://github.com/nodejs/node/blob/084d761dfc1aa9ca0549e7c4d769e30b34347f69/src/node_errors.cc#L917-L923) which prioritizes `ENOTSUP` over `EOPNOTSUPP`.